src/history/HistoryArchive.h

```diff
@@ -96,16 +98,34 @@ struct HistoryArchiveState
     void
     serialize(Archive& ar)
     {
-        ar(CEREAL_NVP(version), CEREAL_NVP(server), CEREAL_NVP(currentLedger),
-           CEREAL_NVP(currentBuckets));
+        ar(CEREAL_NVP(version), CEREAL_NVP(server), CEREAL_NVP(currentLedger));
+        try
+        {
+            ar(CEREAL_NVP(networkPassphrase));
+        }
+        catch (...)
```

Comment:
do not use `catch(...)` but instead catch the exception that you expect in this case (security issue)

Reply:
I think it's `cereal::Exception` here

Reply:
Done!